sdk/ml/azure-ai-ml/azure/ai/ml/automl/_automl_image.py

```diff
@@ -85,7 +85,7 @@ def image_classification(
             Acceptable values: accuracy, AUC_weighted, norm_macro_recall, average_precision_score_weighted,
             and precision_score_weighted
             Defaults to accuracy.
-    :type primary_metric: Union[str, ClassificationPrimaryMetrics]
+    :type primary_metric: Union[str, azure.ai.ml.automl.ClassificationPrimaryMetrics]
```

Comment:
<img width="541" alt="image" src="https://github.com/Azure/azure-sdk-for-python/assets/38021489/2b438876-f5da-4f4e-9b84-0232ae4ab6bc">
For some reason the type here doesn't seem to be showing up...maybe need to use `:paramtype` for keyword arguments?

Reply:
Thanks for catching that. Fixed